### PR TITLE
feature/68/sidebar

### PR DESCRIPTION
**Pull request by tarasSanotskyi**
Create sidebar component 
test sidebar 
refactor navbar
test navbar

image

### CODE REVIEW

File: src/pages/guest-home-page/GuestHome.js [hunk 1]
```diff
@@ -13,12 +13,15 @@ const GuestHomePage = () => {
...
   }
-  
+
   return (
     <Box>
       <button onClick={ handleModal }>click</button>
       <WelcomeBlock />
-      <AccordionWithImage items={ descriptionTimes }  />
+      <AccordionWithImage items={ descriptionTimes } />
+      <Box id='whatCanYouDo' sx={ { width: '100%', height: '100vh', backgroundColor: 'primary.600' } }  />
```
Comment (tarasSanotskyi) [inline src/pages/guest-home-page/GuestHome.js:22]: for functional testing
will be deleted

File: src/containers/navbar/NavBar.js [hunk 2]
```diff
@@ -1,58 +1,33 @@
...
-  }
-}
-
 const Navbar = ({ navigationItems, children }) => {
 
   const { t } = useTranslation()
-  
+  const [isOpen, setIsOpen] = useState(false)
+
+  const openSidebar = () => setIsOpen(true)
```
Comment (maarekua) [inline src/containers/navbar/NavBar.js:19]: I suggest creating one method and passing neither you want to set it to true or false, or rename state ti isSideBarActive, setIsSidebarActive, and use directly setter without methods.

File: src/containers/sidebar/sidebar.style.js [hunk 3]
```diff
@@ -0,0 +1,14 @@
+export const style = {
+  closeIcon: {
+    display: 'flex',
+    justifyContent: 'end',
+    margin: '10px 10px 10px 261px'
```
Comment (maarekua) [inline src/containers/sidebar/sidebar.style.js:5]: margin 261?

File: tests/unit/hooks/use-hash-scroll.spec.js [hunk 4]
```diff
@@ -0,0 +1,40 @@
+import { act, renderHook } from '@testing-library/react-hooks'
+import { screen, render } from '@testing-library/react'
+import useHashScroll from '~/hooks/use-hash-scroll'
+
+jest.mock('react-router-dom', () => ({
```
Comment (maarekua) [inline tests/unit/hooks/use-hash-scroll.spec.js:5]: https://testing-library.com/docs/example-react-router/

**Review by maarekua — CHANGES_REQUESTED** [review]

**Review by maarekua — CHANGES_REQUESTED** [review]

**Review by maarekua — APPROVED** [review]

**Pull request merged**